### DISCUSSION TITLE
EffectTime: Make videos for beginners, we are developers too

### DISCUSSION THREAD

**Discussion by enheit** [Effect-TS/effect-time-podcast#2] [Ideas]
First of all, I want to say thank you for the existing videos in the EffectTime series and add my two cents on what I would like to see in future videos.

I can guess that most applications in the JavaScript world are written using an OO (object oriented) approach due to the popularity of frameworks that extend this style (BE or FE). This means that most developers will try to find something in common between their existing knowledge and approaches to writing apps and something that offers effect-ts.

Since I'm one of those developers trying to integrate effect-ts into my project (backend), I want to see as many examples as possible to solidify my knowledge of how to write code.

I'm not asking you to make a series of tutorials and explain what "monad", "monoid", "functor", etc. are. I think there are enough examples and explanations on the Internet about this, rather I'm asking you to show how it works. How it plays together and how to achieve complex data-flows from the smallest one (with errors, optional values in one flow). How do we need to think in order to write better code with effect-ts.

There are currently several videos in this series that attempt to show actual usage of the library (e.g. [Creating a wrapper around the Node.js filesystem](https://www.youtube.com/watch?v=yEMMI5YYhX8&ab_channel=EFFECT)), but I think (IMHO) the entry point is a bit high.

Also I would like to ask for a few improvements for future videos such as:
- Increase the font in the editor. There are a lot of people who watch videos on their phones, and it's not easy to follow an hour long video with small text.
- Increase webcam frame size. I can understand that the webcam is not the primary element of the scene, but usually you discuss the code for (10-15 seconds) and nothing changes on the screen, intuitively you switch your gaze to the webcam, but the frame is too small to look at it without squinting

Summary

1. Many developers think OOP and try to start thinking in FP for the first time.
2. Existing videos are good, but they are not easy to understand and apply the knowledge gained in your own application.
3. Knowledge about "Monads", "Functors" and other things can be gleaned from other places, they are not critical
4. No knowledge of how to make it work all together (using effect-ts) - critical
5. Increase the font in the editor during the presentation
6. Increase the webcam frame size during the presentation

**Comment by jasonkuhrt** [comment]
Riffing off the title of this comment, a video that starts from scratch and shows how make some kind of small application would be great.

A CLI that does a bunch of IO (e.g. automated package publishing based on looking at git history) would be great, as networked services (APIs) add a bunch more complexity that I think is not needed for an introduction. The OS provides plenty of IO opportunity without dragging in the internet.
